src/components/MapView.tsx

```diff
@@ -1,39 +1,75 @@
-import * as Location from 'expo-location'
+import * as CurrentLocation from 'expo-location'
 import React, { useEffect, useState } from 'react'
-import { StyleSheet } from 'react-native'
+import { StyleSheet, Text } from 'react-native'
 import Map from 'react-native-maps'
-import SearchBar from '@components/SearchBarView'
+import { Business, Location } from '@futureproof/typings'
+import { Marker } from "react-native-maps"
+import { gql, useQuery } from '@apollo/client'
+import Pin from '@components/Pin'
+import DistanceRadiusView from './DistanceRadiusView'
```

Comment:
Use @components again (I don't know if this was you?) Cam A if moving a lot of the files around to needs to be changed.